modules/candidate_profile/jsx/CandidateInfo.js

```diff
@@ -0,0 +1,183 @@
+     * similar to the candidate dashboard.
+     *
+     * @param {array} visits - List of visits in the format returned by the LORIS API.
+     *
+     * @return {object} - A React element containing a comma separated list of links.
+     */
+    getVisitList(visits) {
+        let visitlinks = visits.map( (visit) => {
+            const sessionID = this.props.VisitMap[visit.Meta.Visit];
+            const candID = this.props.Candidate.Meta.CandID;
+            return <a
+                href={
+                    this.props.BaseURL
+                    + '/instrument_list/?candID=' + candID
+                    + '&sessionID=' + sessionID}
+                key={visit.Meta.Visit}
+                >{visit.Meta.Visit}</a>;
+        });
+        return <div>
+            {
+                // Equivalent of .join(', ') that doesn't convert the React
+                // element into the string [object Object].
+                // See https://stackoverflow.com/questions/33577448/is-there-a-way-to-do-array-join-in-react
+                visitlinks.reduce(
+                    (acc, el) => {
+                        if (acc === null) {
+                            return [el];
+                        }
+                        return [acc, ', ', el];
+                    },
+                    null,
+                )
+            }
+            </div>;
+    }
+
+    /**
+     * Render the React component
+     *
+     * @return {object} - The rendered react component
+     */
+    render() {
+        const subprojects = this.getSubprojects(this.props.Visits);
+        const subprojlabel = subprojects.length == 1 ? 'Subproject'
+            : 'Subprojects';
+
+        const data = [
+            {
+                label: 'PSCID',
+                value: this.props.Candidate.Meta.PSCID,
+            },
+            {
+                label: 'DCCID',
+                value: this.props.Candidate.Meta.CandID,
+            },
+            {
+                label: 'Date of Birth',
+                value: this.props.Candidate.Meta.DoB,
+            },
+            {
+                label: 'Age',
+                value: this.calcAge(this.props.Candidate.Meta.DoB),
+            },
+            {
+                label: 'Sex',
+                value: this.props.Candidate.Meta.Sex,
+            },
+            {
+                label: 'Project',
+                value: this.props.Candidate.Meta.Project,
+            },
+            {
+                label: subprojlabel,
+                value: subprojects,
+            },
+            {
+                label: 'Site',
+                value: this.props.Candidate.Meta.Site,
+            },
+            {
+                label: 'Visits',
+                value: this.getVisitList(this.props.Visits),
+                flexBasis: '40%',
+            },
+        ];
+
+        const cardInfo = data.map((info, index) => {
+            const cardStyle = {
+                flexBasis: info.flexBasis || '20%',
```

Comment:
I'm not sure I understand how changing this to 25% would fix things.

Reply:
wouldn't it render the data in 4 columns instead of 5? so then at least in the screenshot example above, it would render 4,4,1 instead of 5,3,1

Reply:
I think we want it to be dynamic based on the size of the card, not a fixed grid (otherwise it should be grid and not the flexbox that we're using..) I've converted it to a font relative unit instead of a container percentage relative unit, can you see it you like this better at a variety of screen sizes?